nxcomp/src/Loop.cpp

```diff
@@ -13491,15 +13492,8 @@ int ParseArg(const char *type, const char *name, const char *value)
     return -1;
   }
 
-  char *string = new char[strlen(value)];
-
-  strncpy(string, value, strlen(value) - 1);
```

Comment:
No, dropping that is not equivalent. The idea is to drop the trailing character and only retain the number.

Probably replace it with

```C++
  char *string = new char[strlen(value)];
  snprintf(string, strlen(value), "%s", value);
```

and keep using `string`.

Reply:
good spot! Fixed!